Sources/TracingInstrumentation/Span.swift

```diff
@@ -77,6 +83,18 @@ extension Span {
     public mutating func addLink(_ parent: Span, attributes: SpanAttributes = [:]) {
         self.addLink(SpanLink(context: parent.context, attributes: attributes))
     }
+
+    public mutating func recordError(ofType type: String?, message: String?) {
+        guard type != nil || message != nil else { return }
+        var attributes = SpanAttributes()
+        if let type = type {
+            attributes["exception.type"] = .string(type)
+        }
+        if let message = message {
+            attributes["exception.message"] = .string(message)
```

Comment:
Makes me think we should NOT implement this to be honest.

Specifically because maybe you're in a system that expects exception, but maybe you're in a system that is more swifty and expects `error.type`.

Swift is not really like the other languages in here: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md as in that errors are not really exceptions, we dont have stacktraces etc...

Makes me feel we should not make that decision; a recorder/tracer should, and it can allow configuring "record errors as "error." or "exception." etc"

WDYT?

Reply:
Yep, sounds reasonable 👍

Reply:
Ok, let's remove the impl then thx